src/components/data-exploration/cell-sets-tool/CellSetsTool.jsx

```diff
@@ -230,7 +230,7 @@ const CellSetsTool = (props) => {
         position: 'relative',
         height: `${height - 40}px`,
         width: `${width - 8}px`,
-        overflow: 'scroll',
```

Comment:
what is the difference when this is in auto?

Reply:
`overflow: 'scroll'` will show the scroll bars even though it is not required. '`overflow: 'auto'` only show the scroll bars when it is required.

![image](https://user-images.githubusercontent.com/2862362/163991586-792d89ff-84d5-40eb-b7fe-1c36fac3ebbc.png)